guides/misc-retrieval-pipeline.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Create a Retrieval Pipeline
+description: Build a LLM retrieval pipeline with Metal (RAG)
+---
+
+## 1. Create a Datasource
```

Comment:
Should we add a bit of explanation around what RAG means?

Reply:
done!